.github/workflows/app-react.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [14.19.3]
```

Comment:
@Fosol just curious if we don't specify the node version does it create an issue with our github workflow?

Reply:
The github action was failing so I had to specify what worked locally.